src/proto/grpc/testing/control.proto

```diff
@@ -129,19 +164,31 @@ message ClientArgs {
 }
 
 message ServerConfig {
-  ServerType server_type = 1;
```

Comment:
removing server_type that also diffentiates between the protobuf server and generic server feels very wrong.